src/Microsoft.OpenApi.Readers/Services/OpenApiReferenceResolver.cs

```diff
@@ -89,6 +89,23 @@ public override void Visit(OpenApiResponses responses)
             ResolveMap(responses);
         }
 
+        /// <summary>
+        /// Resolve all references to SecuritySchemes
```

Comment:
the comments should be "SecurityRequirement" not "SecuritySchemes" ?

Reply:
Actually no.  SecurityRequirement is a dictionary whose keys are SecuritySchemes.  It is those keys that are being replaced if they are unresolved references.